.config-git-ref

```diff
@@ -1 +1 @@
-e99bfc44ff4d66e70eef1be61363887d383a8aa7
\ No newline at end of file
+972fb360bc500ebee92b5155b407f3eabaf44d29
```

Comment:
What's this?

Reply:
git commit sha for the config version to pull from the fr-platform-config repo:
https://github.com/SecureApiGateway/fr-platform-config/pull/4/commits/972fb360bc500ebee92b5155b407f3eabaf44d29